packages/market-service/src/foxy/foxyMockData.ts

```diff
@@ -1,18 +1,12 @@
-import { CoinCapMarketCap } from '../coincap/coincap-types'
+import { CoinGeckoMarketData } from '../coingecko/coingecko-types'
 
-export const fox: CoinCapMarketCap = {
-  id: 'fox-token',
-  rank: '327',
-  symbol: 'FOX',
-  name: 'FOX Token',
-  supply: '272087306.9154830000000000',
-  maxSupply: '1000001337.0000000000000000',
-  marketCapUsd: '76043211.3383411704757409',
-  volumeUsd24Hr: '809217.5632748945831009',
-  priceUsd: '0.2794809217688426',
-  changePercent24Hr: '2.8107676052084740',
-  vwap24Hr: '0.2717415151864480',
-  explorer: 'https://etherscan.io/token/0xc770eefad204b5180df6a14ee197d99d808ee52d'
+export const fox: CoinGeckoMarketData = {
+  circulating_supply: 272087306.915483,
+  max_supply: 1000001337.0,
+  // eslint-disable-next-line @typescript-eslint/no-loss-of-precision
```

Comment:
We should aim to always explain why we disable lint by adding a comment in line.

I assume this is because it's test data and its runtime value doesn't matter.